### ISSUE TITLE
[FEATURE] change entire block

### ISSUE THREAD

**Issue by hisamafahri** [vipzhicheng/logseq-plugin-vim-shortcuts#58]
I hope `cc` in the normal mode, will delete the current block (just like it deletes the current line in vim) and go straight to the `INSERT` mode.

> NOTE: I would be very much happy to help develop this feature, but couldn't find any guides (CONTRIBUTING.md or some sort) to get started

**Comment by vipzhicheng** [comment]
Yes, this has been added by @primeapple , and the shortcut is `d c`， because Logseq uses the c c shortcut.
